interpretation/ww_head.py

```diff
@@ -212,8 +212,16 @@ def process(message, isGameMaster = False):
         # This command removes a given user from a conspiracy channel.
         # A user should not get removed if they're the channel owner.
         if is_command(message,['remove']):
-            # TODO
-            return todo()
+            members_to_remove = check.users(message)
+            if is_owner(user_id,channel_id) == False:
+                return [Mailbox().respond("I\'m sorry, but you cannot use this command over here!")]
+            command = Mailbox()
```

Comment:
Necessary?

Reply:
It should be, considering that only the owner should be able to remove users from a cc.